src/create-model.jl

```diff
@@ -217,6 +220,17 @@ function create_model(graph, representative_periods, dataframes; write_lp_file =
     Fi = filter_flows(:investable, true)
     Ft = filter_flows(:is_transport, true)
 
+    # Create subsets of storage assets by storage type
+    As_short = intersect(As, filter_assets(:storage_type, "short"))
+    As_long  = intersect(As, filter_assets(:storage_type, "long"))
+
+    # Check if all assets in As are in As_short and As_long, if not assign the missing ones to short by default
+    all_storage_assets_in_short_and_long = all(a -> a ∈ union(As_short, As_long), As)
+    if !all_storage_assets_in_short_and_long
+        As_short = union(As_short, setdiff(As, union(As_short, As_long)))
+    end
```

Comment:
This means that some storage types won't be "short" nor "long". Is that allowed? If so, ignore this comment, otherwise, we should create a check preventing this instead of this code.

Reply:
Also, it looks like we could define

```julia
As_long = ... # as it is
As_short = setdiff(As, As_long)
```
i.e., the "long" ones are in `As_long` and the others ("short" or anything else) will be in `As_short`.

Reply:
This is much simpler and easy to read. Thanks! I've changed it.